src/routes/transactions/__tests__/resources/module-transactions/module-transaction-domain.json

```diff
@@ -1,14 +1,15 @@
 {
-    "created": "2022-12-14T13:19:12Z",
-    "executionDate": "2022-12-14T13:19:12Z",
-    "blockNumber": 8133661,
-    "isSuccessful": true,
-    "transactionHash": "0xf4d5521a554f9a889ba4fd642265ed9688405752f8c2574eca05d325487c0b44",
-    "safe": "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C",
-    "module": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
-    "to": "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C",
-    "value": "39900000000000000",
-    "data": null,
-    "operation": 0,
-    "dataDecoded": null
-}
\ No newline at end of file
+  "created": "2022-12-14T13:19:12Z",
+  "executionDate": "2022-12-14T13:19:12Z",
+  "blockNumber": 8133661,
+  "isSuccessful": true,
+  "transactionHash": "0xf4d5521a554f9a889ba4fd642265ed9688405752f8c2574eca05d325487c0b44",
+  "safe": "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C",
+  "module": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+  "to": "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C",
+  "value": "39900000000000000",
+  "data": null,
+  "operation": 0,
+  "dataDecoded": null,
+  "moduleTransactionId": "i5a6754140f0432d3b5e6d8341597ec3c5338239f8d311de9061fbc959f443d590"
+}
```

Comment:
Besides including `moduleTransactionId`, I fixed the indentation and included the newline character at the end of the file.